ansible/roles/test/files/ptftests/inner_hash_test.py

```diff
@@ -479,17 +496,15 @@ def check_balancing(self, dest_port_list, port_hit_cnt):
             total_entry_hit_cnt = 0
             for member in ecmp_entry:
                 total_entry_hit_cnt += port_hit_cnt.get(member, 0)
-            (p, r) = self.check_within_expected_range(total_entry_hit_cnt, float(total_hit_cnt)/len(dest_port_list))
+
+            total_expected = float(total_hit_cnt) / len(dest_port_list)
+            if self.symmetric_hashing and hash_key != 'ip-proto':
+                total_expected = total_expected * 2
```

Comment:
Why is the total expected doubled for symmetric hashing, how was the test working in the past if symmetric hashing expects the total expected packets to double?